USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -70,9 +125,24 @@ private string RenderMarker(Marker input)
                         output.Append(RenderMarker(marker));
                     }
                     output.AppendLine("</div>");
+                    output.AppendLine(RenderFootnotes());
+                    // Page breaks after each chapter
+                    output.AppendLine($"{(separateChapters ? "<br class=\"pagebreak\"></br>" : "")}");

```

Comment:
Another case of this should be an if statement instead of a ternary.

Reply:
Are you you saying that appending a new line with an empty string is not efficient, so I should use if statements?

Reply:
The empty string works but the if statement would give far greater performance since the internal array in the string builder wouldn't have to be resized and reallocated.